Terminal.Gui/FileServices/DefaultFileOperations.cs

```diff
@@ -0,0 +1,139 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Abstractions;
+using System.Linq;
+using Terminal.Gui.Resources;
+
+namespace Terminal.Gui.FileServices {
```

Comment:
I know you said you'd put this in a different namespace, but in #2471 I (rightly or wrongly) made the decision to get rid of all sub-namespaces for the public API. My feeling was it just added more complexity with no real value.

I probably should have asked for debate, but I was trying to move quickly and didn't. 

So, do you think there's a big benefit to having `FileServices` be in a distinct namespace?

Reply:
I agree with your change.  I think the less namespaces the better.  I will move everything into the root namespace and rename anything that has ambigious name.  Good naming ensures things cluster anyway even without another namespace e.g. starting all classes with `File...`

Reply:
Fixed in [380e901](https://github.com/gui-cs/Terminal.Gui/pull/2259/commits/380e901e255a660e394e6e83f3101d9954decb8a)